### ISSUE TITLE
webrtc-frame build failed if using node v10 (necessary node version to run start-all.sh)

### ISSUE THREAD

**Issue by andreasunterhuber** [open-webrtc-toolkit/owt-server#593]
Building owt-server (master branch) with node version 10.22.0 crashes during build of `owt-client-javascript` with `webrtc-frame`. See log. How we can fix that? 

```bash
Building addon - webrtc-frame
make: Entering directory '/home/owt-server/source/agent/webrtc/rtcFrame/build'
  CXX(target) Release/obj.target/librtcadapter/../../../core/rtc_adapter/RtcAdapter.o
  CXX(target) Release/obj.target/librtcadapter/../../../core/rtc_adapter/VideoReceiveAdapter.o
  CXX(target) Release/obj.target/librtcadapter/../../../core/rtc_adapter/VideoSendAdapter.o
  CXX(target) Release/obj.target/librtcadapter/../../../core/rtc_adapter/AudioSendAdapter.o
  CXX(target) Release/obj.target/librtcadapter/../../../core/owt_base/SsrcGenerator.o
  CXX(target) Release/obj.target/librtcadapter/../../../core/owt_base/AudioUtilitiesNew.o
  CXX(target) Release/obj.target/librtcadapter/../../../core/owt_base/TaskRunnerPool.o
In file included from ../../../../core/rtc_adapter/AdapterInternalDefinitions.h:8:0,
                 from ../../../../core/rtc_adapter/VideoReceiveAdapter.h:8,
                 from ../../../../core/rtc_adapter/VideoReceiveAdapter.cc:5:
/home/owt-server/third_party/webrtc-m79/src/api/task_queue/task_queue_factory.h:15:10: fatal error: absl/strings/string_view.h: No such file or directory
 #include "absl/strings/string_view.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from ../../../../core/rtc_adapter/AdapterInternalDefinitions.h:8:0,
                 from ../../../../core/rtc_adapter/RtcAdapter.cc:5:
/home/owt-server/third_party/webrtc-m79/src/api/task_queue/task_queue_factory.h:15:10: fatal error: absl/strings/string_view.h: No such file or directory
 #include "absl/strings/string_view.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from ../../../../core/rtc_adapter/AdapterInternalDefinitions.h:8:0,
                 from ../../../../core/rtc_adapter/VideoSendAdapter.h:8,
                 from ../../../../core/rtc_adapter/VideoSendAdapter.cc:5:
/home/owt-server/third_party/webrtc-m79/src/api/task_queue/task_queue_factory.h:15:10: fatal error: absl/strings/string_view.h: No such file or directory
 #include "absl/strings/string_view.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from ../../../../core/rtc_adapter/AdapterInternalDefinitions.h:8:0,
                 from ../../../../core/rtc_adapter/AudioSendAdapter.h:8,
                 from ../../../../core/rtc_adapter/AudioSendAdapter.cc:5:
/home/owt-server/third_party/webrtc-m79/src/api/task_queue/task_queue_factory.h:15:10: fatal error: absl/strings/string_view.h: No such file or directory
 #include "absl/strings/string_view.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
librtcadapter.target.mk:176: recipe for target 'Release/obj.target/librtcadapter/../../../core/rtc_adapter/VideoReceiveAdapter.o' failed
make: *** [Release/obj.target/librtcadapter/../../../core/rtc_adapter/VideoReceiveAdapter.o] Error 1
make: *** Waiting for unfinished jobs....
librtcadapter.target.mk:176: recipe for target 'Release/obj.target/librtcadapter/../../../core/rtc_adapter/RtcAdapter.o' failed
librtcadapter.target.mk:176: recipe for target 'Release/obj.target/librtcadapter/../../../core/rtc_adapter/VideoSendAdapter.o' failed
make: *** [Release/obj.target/librtcadapter/../../../core/rtc_adapter/RtcAdapter.o] Error 1
make: *** [Release/obj.target/librtcadapter/../../../core/rtc_adapter/VideoSendAdapter.o] Error 1
librtcadapter.target.mk:176: recipe for target 'Release/obj.target/librtcadapter/../../../core/rtc_adapter/AudioSendAdapter.o' failed
make: *** [Release/obj.target/librtcadapter/../../../core/rtc_adapter/AudioSendAdapter.o] Error 1
In file included from /home/owt-server/third_party/webrtc-m79/src/rtc_base/critical_section.h:14:0,
                 from ../../../../core/owt_base/SsrcGenerator.h:10,
                 from ../../../../core/owt_base/SsrcGenerator.cc:5:
/home/owt-server/third_party/webrtc-m79/src/rtc_base/checks.h:45:10: fatal error: absl/meta/type_traits.h: No such file or directory
 #include "absl/meta/type_traits.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
librtcadapter.target.mk:176: recipe for target 'Release/obj.target/librtcadapter/../../../core/owt_base/SsrcGenerator.o' failed
make: *** [Release/obj.target/librtcadapter/../../../core/owt_base/SsrcGenerator.o] Error 1
make: Leaving directory '/home/owt-server/source/agent/webrtc/rtcFrame/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Linux 4.19.76-linuxkit
gyp ERR! command "/usr/local/bin/node" "/usr/local/bin/node-gyp" "rebuild" "-j" "10" "--loglevel=error"
gyp ERR! cwd /home/owt-server/source/agent/webrtc/rtcFrame
gyp ERR! node -v v10.22.0
gyp ERR! node-gyp -v v6.1.0
gyp ERR! not ok
child_process.js:669
    throw err;
    ^

Error: Command failed: node-gyp rebuild -j 10 --loglevel=error
    at checkExecSyncError (child_process.js:629:11)
    at execSync (child_process.js:666:13)
    at buildTarget (/home/owt-server/scripts/build.js:130:5)
    at buildList.map (/home/owt-server/scripts/build.js:161:10)
    at Array.map (<anonymous>)
    at Object.<anonymous> (/home/owt-server/scripts/build.js:156:23)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
The command '/bin/sh -c cd /home && git clone --depth=1 -b ${OWT_BRANCH} ${OWT_SDK_REPO} && cd owt-client-javascript/scripts && npm install && grunt  &&     mkdir ${SERVER_PATH}/third_party/quic-lib &&     export LD_LIBRARY_PATH=/usr/local/lib/x86_64-linux-gnu &&     cd ${SERVER_PATH}/third_party/quic-lib && wget https://github.com/open-webrtc-toolkit/owt-deps-quic/releases/download/v0.1/dist.tgz && tar xzf dist.tgz &&     cd ${SERVER_PATH} && export PKG_CONFIG_PATH=/usr/local/lib/x86_64-linux-gnu/pkgconfig && ./scripts/build.js -t mcu-all -r -c &&     ./scripts/pack.js -t all --install-module --no-pseudo --app-path /home/owt-client-javascript/dist/samples/conference' returned a non-zero code: 1
```

**Why** I'm using not version `v8.15.0` of node? Because startup script requires node 10 and the webrtc_agent crashes if I just upgrade afterwards to the new node version `v10.22.0` (crash telling me it was build with another node version, I should recompile). Connected to https://github.com/open-webrtc-toolkit/owt-server/issues/590

**Comment by starwarfan** [comment]
Hi,
You may need re-install some dependencies after the update.
Run `scripts/installDeps.sh` to perform a full install.

**Comment by andreasunterhuber** [comment]
@starwarfan at the end it was a misleading entry in `/owt-server/dist/package.json`. I changed the node version in the file from `10.21.0` to `8.15.0` and it worked with all depencies. 

Command: `sed -i 's/10.21.0/8.15.0/g' /home/owt-server/dist/package.json` 

Should be fixed anyway: because the version `10.21.0` of node was never actively used!

**Comment by andreasunterhuber** [comment]
Fixed by updating some files. Preparing a pull request for fix. Connected to: https://github.com/open-webrtc-toolkit/owt-server/issues/602
